src/backend/api/reviews.js

```diff
@@ -0,0 +1,58 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+router.get("/reviews", async (request, response) => {
+    try {
+        const reviews = await knex("reviews").select("*");
+        response.json(reviews);
+    } catch (error) { response.status(500).json({ error: "An unexpected error occurred while processing your request." }); }
+});
+
+
+router.post("/reviews", async (request, response) => {
+    const body = request.body;
+    body.created_date = new Date();
+
+    try {
```

Comment:
I would recommend wrapping the entire function in the try block, just to be sure that regardless of what happens, the user will get a meaningful error message.